databricks/koalas/plot.py

```diff
@@ -300,6 +300,7 @@ def _plot(self, ax, bxpstats, column_num=None, return_type="axes", **kwds):
 
     def _compute_plot_data(self):
         colname = self.data.name
```

Comment:
Maybe I think we can remove `colname` by changing the variable name below?

```diff
     def _compute_plot_data(self):
-        colname = self.data.name
         spark_column_name = self.data._internal.spark_column_name_for(self.data._column_label)
         data = self.data

@@ -309,7 +308,7 @@ class KoalasBoxPlot(BoxPlot):
         # Gets some important kwds
         showfliers = self.kwds.get("showfliers", False)
         whis = self.kwds.get("whis", 1.5)
-        labels = self.kwds.get("labels", [colname])
+        labels = self.kwds.get("labels", [spark_column_name])
```

Reply:
Nope, `labels` should be the user-facing label. This is shown in the output plot image for example.

Reply:
@HyukjinKwon 
Oh, Sorry I should have checked the visual output on the notebook not only unittest.
Thanks for the correction !